bin/Audio_Transcript.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import os
+import glob
+import webrtcvad
+import logging
+import waveSplit
+from deepspeech.model import Model
+import scipy.io.wavfile as wav
+from timeit import default_timer as timer
+
+
+# Debug helpers
+logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
+
+
+def main(args):
+    if len(args) != 2:
+        sys.stderr.write(
+            'Usage: example.py <aggressiveness> <path to wav files>\n')
+        sys.exit(1)
+
+    N_FEATURES = 26
+    N_CONTEXT = 9
+    BEAM_WIDTH = 500
+    LM_WEIGHT = 1.75
+    VALID_WORD_COUNT_WEIGHT = 1.00
+    WORD_COUNT_WEIGHT = 1.00
+
+    # Run deepspeech on the directory containing wav files
+    dirName = input("Enter the deepspeech models directory path: ")
```

Comment:
There's still duplication between the GUI and and the CLI interfaces on those, it'd be better you refactor them :)

Reply:
Yup !! Will take care of this..